Controller/ConnectController.php

```diff
@@ -257,7 +257,11 @@ public function connectServiceAction(Request $request, $service)
      */
     public function redirectToServiceAction(Request $request, $service)
     {
-        $authorizationUrl = $this->container->get('hwi_oauth.security.oauth_utils')->getAuthorizationUrl($request, $service);
+        try {
+            $authorizationUrl = $this->container->get('hwi_oauth.security.oauth_utils')->getAuthorizationUrl($request, $service);
+        } catch (\RuntimeException $e) {
+            throw new NotFoundHttpException($e->getMessage(), $e);
```

Comment:
Fixes issue:  Invalid Resource owner errors do not throw CRITICAL error

Exception was not caught. We just rethrow as 404.